### PR TITLE
Add LVMs LLaVA component

### PR DESCRIPTION
**Pull request by Spycsh**
## Description

Add LVMs LLaVA component

## Issues

n/a

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

None
## Tests

tests/test_lvm.sh

### CODE REVIEW

**Review by lvliang-intel — APPROVED** [review]

File: comps/cores/mega/constants.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ class ServiceType(Enum):
     DATAPREP = 9
     UNDEFINED = 10
     RAGAS = 11
+    Img2txt = 12
```
Comment (hshen14) [inline comps/cores/mega/constants.py:29]: uppercase?

**Review by hshen14 — COMMENTED** [review]
Shall we call "Img2txt" to "LVM"?  Image to text sounds too specific.

LVM means large visual model.

File: comps/cores/mega/constants.py (hunk 1 above)
Comment (Spycsh) [reply comps/cores/mega/constants.py]: fixed

**Comment by Spycsh** [comment]
> Shall we call "Img2txt" to "LVM"? Image to text sounds too specific.
> 
> LVM means large visual model.

Hi@hshen14 The original design here is to not only use LLaVA as the base model which convert image+prompt => text, but also adaptable to use image-captioning models such as [vit](https://huggingface.co/nlpconnect/vit-gpt2-image-captioning), [blip](https://huggingface.co/Salesforce/blip-image-captioning-large), which simply convert image=>text. So here I think img2txt is a more general term. What do you think?

**Review by chensuyue — APPROVED** [review]

**Review by XinyuYe-Intel — APPROVED** [review]

**Pull request merged**